src/elements/content-explorer/stories/tests/ContentExplorer-visual.stories.js

```diff
@@ -15,6 +15,12 @@ export const basic = {
         expect(await canvas.findByText('An Ordered Folder')).toBeInTheDocument();
         expect(canvas.getByText('Tue Apr 16 2019 by Preview')).toBeInTheDocument();
 
+        expect(canvas.getByText('Archive')).toBeInTheDocument();
+        expect(canvas.getByText('Wed Dec 16 2020 by Preview')).toBeInTheDocument();
+
+        expect(canvas.getByText('Archived Folder')).toBeInTheDocument();
+        expect(canvas.getByText('Thu Dec 17 2020  by Preview')).toBeInTheDocument();
+
```

Comment:
I know we discussed adding it in new test, but since I've had to add them to mocked root folder (or I could've put them in subfolder, but that would require to click into subfolder and verify they're there) I've decided to just check in this place

Reply:
fyi this file refers to the element version of content explorer. but the feature you added was for the "features" version of content explorer. i.e. these are two different components that do not share logic